pom.xml

```diff
@@ -633,7 +633,13 @@
 		    <artifactId>jira-rest-java-client</artifactId>
 		    <version>2.0.0-m2</version>
 		</dependency>
-	</dependencies>
+		<dependency>
+			<groupId>com.googlecode.gwtquery</groupId>
```

Comment:
How much does this add to the client download (the gwt wrapper for jquery)?  If we can incorporate (to avoid "callback hell" by using promises), maybe it's worth it.

Reply:
Is it well maintained, does it lock us into a version of jquery that it wraps?